api/v1/tenant_types.go

```diff
@@ -66,8 +70,14 @@ type TenantSpec struct {
 	// +required
 	ID string `json:"id,omitempty"`
 
+	// Name is a human readable name for this tenant.
+	Name string `json:"name,omitempty"`
+
 	// Indices defines the how to store a tenant's data
 	Indices []Index `json:"indices"`
+
+	// Elastic configures per-tenant ElasticSearch and Kibana parameters.
+	Elastic *TenantElasticSpec `json:"elastic,omitempty"`
```

Comment:
When can this field be missing ?

Reply:
This field can be omitted if not using an external elasticsearch. I'll add a comment